plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -114,6 +114,31 @@ void validate(boost::any& v,
   }
 }
 
+void validate(boost::any& v,
+              const std::vector<std::string>& values,
+              wasm_interface::vm_oc_enable* /* target_type */,
+              int)
+{
+  using namespace boost::program_options;
+
+  // Make sure no previous assignment to 'v' was made.
+  validators::check_first_occurrence(v);
+
+  // Extract the first string from 'values'. If there is more than
+  // one string, it's an error, and exception will be thrown.
+  std::string const& s = validators::get_single_string(values);
+
+  if (s == "auto") {
+     v = boost::any(wasm_interface::vm_oc_enable::oc_auto);
+  } else if (s == "all" || s == "true" || s == "on" || s == "yes" || s == "1") {
+     v = boost::any(wasm_interface::vm_oc_enable::oc_all);
+  } else if (s == "none" || s == "false" || s == "off" || s == "no" || s == "0") {
```

Comment:
fwiw, it appears boost was doing a case insensitive compare; so `False`, `oN`, etc was also accepted :smiling_imp: depending on how accommodating you want to be...